starknet_devnet/starknet_wrapper.py

```diff
@@ -36,9 +36,13 @@
     StateDiff,
     StorageEntry,
 )
+from starknet_devnet.util import to_bytes
```

Comment:
Can you change to use the one imported from starkware?

Reply:
done